Kube/scripts/deploy-shield.sh

```diff
@@ -119,7 +123,10 @@ while [ $# -ne 0 ]; do
          exit
         else
          shift
-         only_namespace "$1"
+         while [ ! -z "$1" ] && [[ "$1" == *"shield-"* ]] ; do
```

Comment:
traversing on input of namespaces,
check if there is another arg passed, if so checks if it is a shield namespace (by comparing to substring *shield-*)

Reply:
by doing that you are "eating" the last argument
so for ex:
./deploy.sh  -n shield-management shield-proxy -l

will not do the -l (because of the shift)
you can work on the $2 argument:
<pre><code>
-n | --namespace)
       if [ -z "$2" ]; then
         echo "missing namespace"
         usage
         exit
        else
         while [ ! -z "$2" ] && [[ "$2" == *"shield-"* ]] ; do
            only_namespace "$2"
            shift
         done
        fi
        ;;
</pre><c/ode>